src/pages/studyView/charts/mutationPlot/StudyViewMutationMapperStore.ts

```diff
@@ -0,0 +1,66 @@
+import { Mutation } from 'cbioportal-ts-api-client';
+import { DefaultMutationMapperStoreConfig } from '../../../../../packages/react-mutation-mapper/dist/store/DefaultMutationMapperStore';
```

Comment:
This is not the right way to import from packages. Do this instead:

```suggestion
import { DefaultMutationMapperStoreConfig } from 'react-mutation-mapper';
```